staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -66,6 +66,11 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 					{Name: "Age", Type: "date", Description: swaggerMetadataDescriptions["creationTimestamp"], JSONPath: ".metadata.creationTimestamp"},
 				}
 			}
+			if len(obj.SelectableFields) == 0 {
+				obj.SelectableFields = []apiextensions.SelectableField{
+					{JSONPath: ".spec.example"},
```

Comment:
why can't we let this field fuzz as normal for round tripping?

Reply:
My understanding is that it's because of the `c.FuzzNoCustom(obj)` at the top of the function. Not sure if that's a satisfactory answer or not..

Reply:
Try using c.Fuzz(obj.SelectableFields) here

Reply:
Works. Thanks!